src/encoding/armor.js

```diff
@@ -246,7 +244,7 @@ function splitChecksum(text) {
 
   if (lastEquals >= 0) {
```

Comment:
How about trim the text and then use 'lastEquals !== text.length - 1'. Unlikely to be a problem, but that way extra whitespace doesn't affect this.